packages/server/src/profile/profile.controller.ts

```diff
@@ -81,4 +95,39 @@ export class ProfileController {
 
     return this.get(user);
   }
+
+  @Post('/upload_picture')
+  @UseInterceptors(
+    FileInterceptor('file', {
+      dest: 'uploads/',
```

Comment:
For compression, we probably don't want to save to file system, get the file, compress, and save to file system again. 

Maybe we switch to the Memory engine and do file compression + saving ourselves.
https://github.com/expressjs/multer#memorystorage

Reply:
Also, use an envionment variable for the file upload location.